crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -507,10 +486,9 @@ where
 
 #[cfg(test)]
 mod tests {
-    use bytes::Bytes;
+    use insta::assert_json_snapshot;
```

Comment:
Doesn't look like any new usage added here?

Reply:
Yeah, that was a strange import after rebasing. Removed.